health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -766,175 +766,176 @@
+             const file = files.find((item) => item.templateIdentifier === "Population");
+             const fileId = file?.filestoreId;
+             if (!fileId) {
+                   console.error("Population template file not found");
+                   return;
+                 }
+             const campaignName = row?.name || "";
+             Digit.Utils.campaign.downloadExcelWithCustomName({
+               fileStoreId: fileId,
+               customName: campaignName
+             });
+           };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)